test/e2e/data/infrastructure-docker/bases/cluster-with-kcp.yaml

```diff
@@ -37,7 +37,10 @@ metadata:
   name: "${CLUSTER_NAME}-control-plane"
 spec:
   template:
-    spec: {}
+    spec:
+      extraMounts:
+        - containerPath: /var/run/docker.sock
+          hostPath: /var/run/docker.socks
```

Comment:
typo?

Reply:
I'm following a lead
It seems that self hosted test are failing when CAPD controller get scheduler on the control plane node because control plane node is missing the mounts for the docker socket...